core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -88,6 +88,12 @@ public boolean accept(File dir,String name) {
         }
     };
 
+    /**
+     * If set this should be the absolute file path to the the base directory for all exploded .hpi/.jpi plugins.
+     * If {@literal null} then plugins will be exploded into {@literal JENKINS_HOME/plugins}.
+     */
+    private static final String WORK_DIR = System.getProperty(ClassicPluginStrategy.class.getName() + ".WORK_DIR");
```

Comment:
Most other system properties in Jenkins are backing a run-time mutable value (i.e. they are **static** but non-final) and in those cases the field should be camelCase to reflect not being a constant. If you read the commit message and the JIRA you will see the logic as to why this cannot be runtime mutable